pydm/widgets/image.py

```diff
@@ -116,17 +120,44 @@ def setColorMapLimits(self, mn, mx):
         self.cm_min = mn
         self.setColorMap()
 
-    def setColorMapToPreset(self, name):
+    @pyqtProperty(PyDMColorMap)
+    def colorMap(self):
+        """
+        The color map used by the ImageView.
+
+        Returns
+        -------
+        PyDMColorMap
+        """
+        return self._colormap
+    
+    @colorMap.setter
+    def colorMap(self, new_cmap):
+        """
+        The color map used by the ImageView.
+
+        Parameters
+        -------
+        new_cmap : PyDMColorMap
+        """
+        self.setColorMapToPreset(new_cmap)
+
+    def setColorMapToPreset(self, cmap):
```

Comment:
Is this method from Qt or PyQt somehow? If not, do you mind changing it to the Pythonic way?